docs/source/api/event_management.rst

```diff
@@ -0,0 +1,11 @@
+Event Management
+================
+
+.. currentmodule::  indico.modules.events.management.controllers
+.. autoclass:: RHManageEventBase
```

Comment:
It's the base class for every single management RH. I think such classes (both RH and WP) deserve documentation.